cmd/broker/main.go

```diff
@@ -134,6 +134,9 @@ type Config struct {
 
 	Quota                               quota.Config
 	QuotaWhitelistedSubaccountsFilePath string
+
+	// temporary FF
+	UseHapForDeprovisioning bool
```

Comment:
I think it would be safer to also here define the default value just in case it's missing from the chart.